log/README.md

```diff
@@ -57,9 +57,11 @@ The checkpoint body is of the form:
 The first 3 lines of the body **MUST** be present in all Checkpoints.
 
 * `<Origin string>` should be a unique identifier for the log identity which issued the checkpoint.
-  The exact format of this is left undefined, but best practice is to have a scheme-less URI-like
-  prefix that will be globally unique, and give some idea of the log location/operator,
-  e.g. `example.com/log42`.
+  The exact format of this is left undefined, but best practice is to use a URI-like strucutre like
```

Comment:
Could you use https://www.ietf.org/rfc/rfc2119.txt language, like "The format SHOULD be a URI-like structure. Clients MUST NOT assume that the origin is a URI"?